kubernetes/resource_kubernetes_job_v1_test.go

```diff
@@ -77,6 +77,13 @@ func TestAccKubernetesJobV1_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "spec.0.template.0.spec.0.container.0.name", "hello"),
 					resource.TestCheckResourceAttr(resourceName, "spec.0.template.0.spec.0.container.0.image", imageName),
 					resource.TestCheckResourceAttr(resourceName, "wait_for_completion", "false"),
+					resource.TestCheckResourceAttr(resourceName, "spec.0.pod_failure_policy.0.rule.#", "2"),
+					resource.TestCheckResourceAttr(resourceName, "spec.0.pod_failure_policy.0.rule.0.action", "FailJob"),
+					resource.TestCheckResourceAttr(resourceName, "spec.0.pod_failure_policy.0.rule.0.on_exit_codes.0.container_name", "hello"),
+					resource.TestCheckResourceAttr(resourceName, "spec.0.pod_failure_policy.0.rule.0.on_exit_codes.0.values.#", "3"),
```

Comment:
Made `values` argument be `Required`. Not including `values` would lead to it being an empty slice. Resulting in the error below.

```
        Error: Failed to create Job! API error: Job.batch "tf-acc-test-3k1whe3ocj" is invalid: spec.podFailurePolicy.rules[0].onExitCodes.values: Invalid value: []int32{}: at least one value is required
        
          with kubernetes_job_v1.test,
          on terraform_plugin_test.tf line 1, in resource "kubernetes_job_v1" "test":
           1: resource "kubernetes_job_v1" "test" {
           ```